### PR TITLE
INT-69 doc extraction

### PR DESCRIPTION
**Pull request by rkaczka-vd**
Contains:
- https://github.com/IndicoDataSolutions/indico-client-csharp/pull/59

### CODE REVIEW

File: IndicoV2.Abstractions/Ocr/IOcrClient.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        /// <param name="cancellationToken"><c><see cref="CancellationToken"/></c> for handling cancellation of asynchronous operations.</param>
+        /// <returns>Job's Is</returns>
+        Task<string> ExtractDocumentAsync(string filePath, string configType = "standard", CancellationToken cancellationToken = default);
+
+        /// <summary>
+        /// Gets result of the OCR
+        /// </summary>
+        /// <param name="documentUri">Url of the document (returned in Job's result).</param>
+        /// <returns>Content of the document</returns>
+        Task<string> GetExtractionResult(Uri documentUri);
```
Comment (szymon-socha) [inline IndicoV2.Abstractions/Ocr/IOcrClient.cs:23]: These names are not consistent. Both are async, one of them has an `Async` suffix.

File: IndicoV2.IntegrationTests/Ocr/OcrClientTests.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            _jobAwaiter = container.Resolve<JobAwaiter>();
+            _ocrClient = container.Resolve<IOcrClient>();
+        }
+
+        [TestCase("standard")]
+        public async Task ExtractDocument_ShouldReturnJobNumber(string configType)
+            => (await _ocrClient.ExtractDocumentAsync(_dataHelper.Files().GetSampleFilePath(), configType, default))
+                .Should().NotBeEmpty();
+
+        [TestCase]
```
Comment (szymon-socha) [inline IndicoV2.IntegrationTests/Ocr/OcrClientTests.cs:36]: Why `TestCase`?

**Review by szymon-socha — APPROVED** [review]

File: IndicoV2.Abstractions/Ocr/IOcrClient.cs (hunk 1 above)
Comment (rkaczka-vd) [reply IndicoV2.Abstractions/Ocr/IOcrClient.cs:23]: fixed

File: IndicoV2.IntegrationTests/Ocr/OcrClientTests.cs (hunk 2 above)
Comment (rkaczka-vd) [reply IndicoV2.IntegrationTests/Ocr/OcrClientTests.cs:36]: fixed

**Pull request merged**